### PR TITLE
GN-161 Skip submenu accessibility

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/GN-161

### CODE REVIEW

File: src/Header/Nav/NavLinks/NavLinks.jsx [hunk 1]
```diff
@@ -128,7 +128,9 @@ export function NavLinks({
 											onClick={handleNavLinkClick}
 											id={`navlink-${id}`}
 										>
-											<span aria-label={abbreviation && title}>{text}</span>
+											<span aria-label={abbreviation && title} id={id}>
```
Comment (ediblecode) [inline src/Header/Nav/NavLinks/NavLinks.jsx:131]: I think we can take the if odd here now can't we?

File: src/Header/Nav/NavLinks/NavLinks.test.jsx [hunk 2]
```diff
@@ -59,6 +59,15 @@ describe("NavLinks", () => {
 		expect(link.props()["aria-current"]).toBe(true);
 	});
 
+	it.only("should add render an abbreviation title element with an id that matches the current service id", () => {
```
Comment (ediblecode) [inline src/Header/Nav/NavLinks/NavLinks.test.jsx:62]: Can you remove the only off here?

File: src/Header/Nav/NavLinks/NavLinks.test.jsx [hunk 3]
```diff
@@ -59,6 +59,15 @@ describe("NavLinks", () => {
...
+		const wrapper = shallow(
+			<NavLinks {...defaultProps} currentService="link2" />
+		);
+		const link = wrapper.find("[id='link2']");
+		expect(link.props()["aria-label"]).toBe("Abbreviation title");
+		expect(link.props()["id"]).toEqual("link2");
```
Comment (ediblecode) [inline src/Header/Nav/NavLinks/NavLinks.test.jsx:68]: If we're removing the id then I think we can remove this test, or at least the assertion

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: src/Header/Nav/NavLinks/NavLinks.jsx (hunk 1 above)
Comment (johndavey72) [reply src/Header/Nav/NavLinks/NavLinks.jsx:131]: ahh forgot there were two :-)

File: src/Header/Nav/__snapshots__/Nav.test.jsx.snap [hunk 4]
```diff
@@ -232,7 +232,9 @@ exports[`Nav internalServices Matches snapshot with sub links for selected inter
                   id="navlink-internal1"
                   onClick={[Function]}
                 >
-                  <span>
+                  <span
```
Comment (ediblecode) [inline src/Header/Nav/__snapshots__/Nav.test.jsx.snap:235]: This snapshot looks wrong to me

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: src/Header/Nav/__snapshots__/Nav.test.jsx.snap (hunk 4 above)
Comment (johndavey72) [reply src/Header/Nav/__snapshots__/Nav.test.jsx.snap]: updated the snapshot and fixed a couple of failing tests

**Review by ediblecode — APPROVED** [review]

**Pull request merged**